gcc/rust/expand/rust-expand-visitor.cc

```diff
@@ -1395,7 +1396,8 @@ template <typename T>
 void
 ExpandVisitor::expand_outer_attribute (T &item, AST::SimplePath &path)
 {
-  // FIXME: Implement outer attribute expansion
+  // FIXME: Retrieve path from segments + local use statements instead of string
+  proc_expander.expand_attribute_proc_macro (item, path);
```

Comment:
Mentioned offline that we should rework this and look at it from a "container's point of view", as in perform the expansion of items directly from the surrounding crate/module.